client/src/components/edit/AddExperience.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
+import { DatePicker, MenuItem } from 'material-ui'
+import PropTypes from 'prop-types'
+
 import { StyledTextField } from '../../widgets/StyledTextField'
 import { AddButton } from '../../widgets/AddButton'
 import { StyledSelectField } from '../../widgets/StyledSelectField'
-import { DatePicker, MenuItem } from 'material-ui'
 import { style } from '../../widgets/styles'
 import './edit.css'
 
-export const AddExperience = props => {
+const AddExperience = props => {
   const { type } = props
```

Comment:
Is there a reason why this syntax is preferred over the previous way the components were written?

Reply:
linter was complaining about the other way